pyanaconda/packaging/dnfpayload.py

```diff
@@ -567,11 +570,23 @@ def _spaceRequired(self):
         if transaction is None:
             return Size("3000 MB")
 
-        size = sum(tsi.installed.installsize for tsi in transaction)
-        # add 35% to account for the fact that the above method is laughably
-        # inaccurate:
-        size *= 1.35
-        return Size(size)
+        size = 0
+        files_nm = 0
+        for tsi in transaction:
+            # space taken by all files installed by the packages
+            size += tsi.installed.installsize
+            # number of files installed on the system
+            files_nm += len(tsi.installed.files)
+        # add required space which depends on block size and rpm database taken space estimation
+        # 6144 = 4K(max default fragment size) + 2K(rpm db could be taken for a header file)
+        bonus_size = files_nm * 6144
```

Comment:
This ``6144`` should be constant defined at the top of the file with appropriate name and, what's more important, it should be ``Size("6 KiB")``.